hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AsyncProtobufLogWriter.java

```diff
@@ -231,4 +231,9 @@ protected long writeWALTrailerAndMagic(WALTrailer trailer, byte[] magic) throws
   protected OutputStream getOutputStreamForCellEncoder() {
     return asyncOutputWrapper;
   }
+
+  @Override
+  public long getSyncedLength() {
+    return this.output.getSyncedLength();
```

Comment:
nit: `this` is unnecessary.

Reply:
Just as above mentioned, because  current PR is backport, after this PR is applied to 2.x, I would open another addnum PR for both master and 2.x to incorporate your suggestions.

Reply:
Yep, understood. Thanks.